src/components/PatchAlbum.jsx

```diff
@@ -0,0 +1,150 @@
+      fetch(`${config.API_URL}/api/albums/${id}`),
+    ];
+    Promise.all(requestPromises)
+      .then(([artistsResponse, albumResponse]) => {
+        if (!artistsResponse.ok) {
+          setError(true);
+          return artistsResponse.text().then((msg) => Promise.reject(new Error(msg)));
+        }
+        if (!albumResponse.ok) {
+          setError(true);
+          return albumResponse.text().then((msg) => Promise.reject(new Error(msg)));
+        }
+        return Promise.all([artistsResponse.json(), albumResponse.json()]);
+      })
+      .then(([artistsData, albumData]) => {
+        new Deserializer({ keyForAttribute: 'camelCase' }).deserialize(artistsData, (_error, artistList) => setArtists(artistList));
+        new Deserializer({ keyForAttribute: 'camelCase' }).deserialize(albumData, (_error, albumInfo) => setAlbum(albumInfo));
+      })
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>{error}</p>;
+  }
+  return (
+    <div className="form">
+      <Formik
+        initialValues={{
+          artistId: album?.artistId,
+          name: album?.name,
+          publishedAt: album?.publishedAt,
+          cover: album?.cover,
+        }}
+        validationSchema={Yup.object({
+          name: Yup.string()
+            .required('This field is required'),
+          artistId: Yup.number()
+            .required('This field is required'),
+          publishedAt: Yup.date()
+            .required('This field is required'),
+          cover: Yup.string(),
+        })}
+        onSubmit={async (values) => {
+          setLoading(true);
+          const formData = new FormData();
+          Object.keys(values).forEach((key) => {
+            formData.append(key, values[key]);
+          });
+          const requestOptions = {
+            method: 'PATCH',
+            headers: {
+              Authorization: `Bearer ${currentUser?.access_token}`,
+            },
+            body: formData,
+          };
+          try {
+            const response = await fetch(`${config.API_URL}/api/albums/${id}`, requestOptions);
+            if (response.status !== 200) {
+              setError(true);
+              const err = await response.text();
+              throw new Error(err);
+            } else {
+              navigate(`/albums/${id}`);
+            }
+          } catch (err) {
+            setMessage(err.message);
+          } finally {
+            setLoading(false);
+          }
+        }}
+      >
+        {({ errors, touched, setFieldValue }) => (
+          <Form>
+            <div>
+              <label htmlFor="artistId">Artist:</label>
+              <Field name="artistId" as="select">
+                {artists.map((artist) => (
+                  <option key={artist.id} value={artist.id}>
+                    {artist.name}
+                    {' '}
+                  </option>
```

Comment:
Hay un espacio extra después del nombre del artista, que creo que se puede eliminar
```suggestion
                  <option key={artist.id} value={artist.id}>
                    {artist.name}
                  </option>
```